Lib/test/test_pydoc.py

```diff
@@ -1250,7 +1250,6 @@ def __get__(self, obj, cls):
         class X:
             attr = Descr()
 
-        text = pydoc.plain(pydoc.render_doc(X.attr))
```

Comment:
The *name* `text` here and twice below is not used, but I am really sure that the function calls not raising are part of the test and should not be removed.  They  may also raise line coverage of branches within the function code.  So without consulting with the author or spending considerable time to understand code and test and coverage, at most change `text` to `_` or just remove it along with `=`.  Adding a comment would be good, perhaps `# Test calls.`

Reply:
This statement can be safely removed. `_get_summary_lines()` calls the same code, but also removes the first two lines from the result.